src/commonMain/kotlin/ehn/techiop/hcert/kotlin/data/GreenCertificate.kt

```diff
@@ -29,10 +28,11 @@ data class GreenCertificate(
     /**
      * For [dateOfBirthString] ("dob"), month and day are optional in eu-dcc-schema 1.2.1,
      * so we may not be able to get a valid [LocalDate] from it.
+     * Be lenient, i.e. strip a timestamp, if it is included
      */
     val dateOfBirth: LocalDate?
         get() = try {
-            LocalDate.parse(dateOfBirthString)
+            LocalDate.parse(dateOfBirthString.substringBefore("T"))
```

Comment:
Why is this not part of a LenientDoBSerializer or something similar?

Reply:
I did decide against that, because the app may need the `dateOfBirthString` as it was encoded, and we don't expect that we can parse e.g. `1970-01` to a valid `LocalDate` anyway.